Kubernetes/windows/debug/starthnstrace.ps1

```diff
@@ -0,0 +1,43 @@
+param(
+    [bool] $Prompt = $true,
+    [string] $EtlFile = "c:\HnsCap.etl"
```

Comment:
Should we keep the file name consistent with starthnstrace.cmd at server.etl given that this will replace the cmd file?

Reply:
Good idea